scripts/reindex_with_types.py

```
@@ -0,0 +1,315 @@
+        if dry_run:
+            logger.info(f"[DRY RUN] Would delete store: {target_store.name}")
+            return True
+
+        # Delete the store with force=True to delete all documents
+        logger.info(f"Deleting store: {target_store.name} (this may take a while)...")
+        client.file_search_stores.delete(
+            name=target_store.name,
+            config={'force': True}
+        )
+        logger.info("Store deleted successfully")
+        return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)